segmentation/utils/spatialorgmarkerexpression_utils.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+import numpy as np
+import scipy
+import statsmodels
+from statsmodels.stats.multitest import multipletests
+
+# Erin's Data Inputs
+cell_array = pd.read_csv("/Users/jaiveersingh/Downloads/SpatialEn"
+                         "richment/granA_cellpheno_CS-asinh-norm_revised.csv")
+marker_thresholds = pd.read_csv("/Users/jaiveersingh/Downloads/Sp"
+                                "atialEnrichment/markerThresholds.csv")
+dist_matrix = np.asarray(pd.read_csv("/Users/jaiveersingh/Documen"
+                                     "ts/MATLAB/distancesMat5.csv",
+                                     header=None))
+
+# pv_cellarray = pd.read_csv("/Users/jaive
+# ersingh/Desktop/tests/pvcelllabel.csv")
+
+# pv_distmat = np.asarray(pd.read_csv("/Us
+# ers/jaiveersingh/Desktop/tests/pvdistmat.csv", header = None))
+
+# pv_cellarrayN = pd.read_csv("/Users/jaiv
+# eersingh/Desktop/tests/pvcellarrayN.csv")
+
+# pv_distmatN = np.asarray(pd.read_csv("/Use
+# rs/jaiveersingh/Desktop/tests/pvdistmatN.csv", header = None))
+
+# randMat = np.random.randint(0,200,size=(60,60))
+# np.fill_diagonal(randMat, 0)
+
+# pv_cellarrayR = pd.read_csv("/Users/jaiveers
+# ingh/Desktop/tests/pvcellarrayR.csv")
+
+
+def spatial_analysis(dist_matrix, marker_thresholds, cell_array):
+    # Setup input and parameters
```

Comment:
add a docstring here that follows the same formats as the other functions in the repo